server/store/datastore/migration/common.go

```diff
@@ -212,6 +212,42 @@ func dropTableColumns(sess *xorm.Session, tableName string, columnNames ...strin
 	return nil
 }
 
+func alterColumnDefault(sess *xorm.Session, table, column, defValue string) error {
+	dialect := sess.Engine().Dialect().URI().DBType
+	switch dialect {
+	case schemas.MYSQL:
+		_, err := sess.Exec(fmt.Sprintf("ALTER TABLE `%s` COLUMN `%s` SET DEFAULT %s;", table, column, defValue))
+		return err
+	case schemas.POSTGRES:
+		_, err := sess.Exec(fmt.Sprintf("ALTER TABLE `%s` ALTER COLUMN `%s` SET DEFAULT %s;", table, column, defValue))
+		return err
+	case schemas.SQLITE:
+		return nil
```

Comment:
SQLite has no way to alter column default value/not null

Reply:
So we would need to do this temp table copy work-around again?

Reply:
That would be only option but do we need that, I added that just to have more database checks and sqlite3 is used probably mostly for small instances, so imho it's not worth the trouble :)

Reply:
https://ci.woodpecker-ci.org/woodpecker-ci/woodpecker is using sqlite 🙈. Would it still work when having the old column type? Otherwise it would be a breaking change IMO so we would need to mention or migrate it.

Reply:
It would still work even without migration, constraints are added to be more on safe side